installer/scripts/omsadmin.sh

```diff
@@ -325,6 +326,35 @@ set_proxy_setting()
     fi
 }
 
+is_scom_port_open()
+{
+    OIFS=$IFS
+# Ignore formatting. It was required for IFS to take new line on both bash and sh
+IFS='
+'
+    for line in `cat $OMI_CONF_FILE | grep httpsport`
+    do
+        echo $line | grep -E '^#.*$' > /dev/null 2> /dev/null
+        is_comment=$?
+        if [ $is_comment -eq 0 ]; then
+            continue
+        fi
+        # 1st uncommented line is checked for port
+        # Replace = and , with space.
+        # e.g httpsport=0,1270 should become httpsport 0 1270
+        m_line=`echo $line | sed "s/[=,]/ /g"`
```

Comment:
As far as I know, we did not ever specify a dependence on `sed`, but we use it in lots of places. I recently made a change to install sed if it's not already installed; you can see the commit here: https://github.com/Microsoft/OMS-Agent-for-Linux/commit/e93aa8926e2f93912dfe0cc09a8ae3e05e42b8e0